news/4781-convert-manual-clone-fixture

```diff
@@ -0,0 +1,19 @@
+### Enhancements
```

Comment:
is this name is auto-generated?? if not.. is the naming convention fixed??

Reply:
Legacy choice, we can change it, but it's currently standardized across all of our rever configurations:

https://github.com/conda/infrastructure/blob/44a3f326c8c7d23438ad601345b0818e10b49eec/.github/sync/rever.xsh#L19-L25

Reply:
really.. its way to long for a file that describes the news.. let's mull over the naming convention for a moment.. see if there is a way to shorten it.. you wont be able to see the full file name..

Reply:
Wait, I'm not sure I understand what we are talking about here

are we talking about the news snippet filename (this is insignificant and only needs to be unique)? the `### Enhancements` header (this is a legacy choice, see prior comment)? or the line length of the news bullet itself on line 19 (not an issue since this is stitched into a Markdown with linewrap)?

Reply:
the name of the file `4781-convert-manula-clone-fixture` that is too long i was thinking.. the news segments works.. the template works fine

Reply:
yeah, this filename won't exist long term, once we do a release all of these snippets are combined into the CHANGELOG.md and deleted, so the naming choice here is meaningless